iml-agent/src/action_plugins/lustre.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::{agent_error::ImlAgentError, lustre};
+use futures::TryFutureExt;
+
+/// Runs lctl with given arguments
+pub async fn lctl(args: Vec<String>) -> Result<String, ImlAgentError> {
+    lustre::lctl(args.iter().map(|s| s.as_ref()).collect())
+        .await
+        .map(|o| String::from_utf8_lossy(&o.stdout).to_string())
+}
+
+/// Executes `mountpoint -q /mnt/something`
+/// It provides no output, but signals the caller with the exit code
+pub async fn is_mounted(mount_point: String) -> Result<bool, ImlAgentError> {
```

Comment:
You can't be sure that the mounted fs is lustre, can you?

Reply:
So, I investigated the Lustre documentation, and sorry, I haven't found the tool to check that it was Lustre mounted to the mount point. The function will return `true` for anything, even flash drive or RAM disks.
One of the solution is `mount + grep`, looks too overcomlicated for me so far.
@utopiabound What do you suggest for checking the mount point?

Reply:
I believe this is properly resolved by https://github.com/whamcloud/integrated-manager-for-lustre/pull/1176

@jgrund correct?

Reply:
@mkpankov yeah, between the data we get from `findmnt` and being able to expect zfs dataset props, we should know about all mounted devices and their backing type.

Reply:
Given the new information into account, I removed this action at all.